src/libs/PersonalDetailsUtils.js

```diff
@@ -123,7 +123,6 @@ function getNewPersonalDetailsOnyxData(logins, accountIDs) {
              * This is done to prevent duplicate entries (upon success) since the BE will return other personal details with the correct account IDs.
              */
             successData[accountID] = null;
-            failureData[accountID] = null;
```

Comment:
I wonder why we did this here... even the comment say we want to prevent dupes **on success**

Reply:
Yeah, it's a bit odd